### ISSUE TITLE
Error Installing Pixelfed setup in windows10

### ISSUE THREAD

**Issue by ER3777** [pixelfed/support#83]
As i cloned the project folder via github and the error throwing at 

Fatal error: Uncaught Error: Class 'Illuminate\Foundation\Application' not found in D:\xampp\htdocs\pixelfed\bootstrap\app.php:14 Stack trace: #0 D:\xampp\htdocs\pixelfed\public\index.php(36).

this error throws as soon as i type command php artisan key:generate.
Any help thanks in advance.

**Comment by trwnh** [comment]
windows isn't officially supported for installation, but did you run composer?

**Comment by ER3777** [comment]
Yes i did run it but i again encountered errors post updating composer as well as you can see the errors below.

Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - The requested PHP extension ext-intl * is missing from your system. Instal
l or enable PHP's intl extension.
  Problem 2
    - laravel/horizon v3.7.2 requires ext-pcntl * -> the requested PHP extension
 pcntl is missing from your system.
    - laravel/horizon v3.7.1 requires ext-pcntl * -> the requested PHP extension
 pcntl is missing from your system.
    - laravel/horizon v3.7.0 requires ext-pcntl * -> the requested PHP extension
 pcntl is missing from your system.
    - laravel/horizon v3.6.1 requires ext-pcntl * -> the requested PHP extension
 pcntl is missing from your system.
    - laravel/horizon v3.6.0 requires ext-pcntl * -> the requested PHP extension
 pcntl is missing from your system.
    - laravel/horizon v3.5.0 requires ext-pcntl * -> the requested PHP extension
 pcntl is missing from your system.
    - laravel/horizon v3.4.7 requires ext-pcntl * -> the requested PHP extension
 pcntl is missing from your system.
    - laravel/horizon v3.4.6 requires ext-pcntl * -> the requested PHP extension
 pcntl is missing from your system.
    - laravel/horizon v3.4.5 requires ext-pcntl * -> the requested PHP extension
 pcntl is missing from your system.
    - laravel/horizon v3.4.4 requires ext-pcntl * -> the requested PHP extension
 pcntl is missing from your system.
    - laravel/horizon v3.4.3 requires ext-pcntl * -> the requested PHP extension
 pcntl is missing from your system.
    - laravel/horizon v3.4.2 requires ext-pcntl * -> the requested PHP extension
 pcntl is missing from your system.
    - laravel/horizon v3.4.1 requires ext-pcntl * -> the requested PHP extension
 pcntl is missing from your system.
    - laravel/horizon v3.4.0 requires ext-pcntl * -> the requested PHP extension
 pcntl is missing from your system.
    - laravel/horizon v3.3.2 requires ext-pcntl * -> the requested PHP extension
 pcntl is missing from your system.
    - laravel/horizon v3.3.1 requires ext-pcntl * -> the requested PHP extension
 pcntl is missing from your system.
    - laravel/horizon v3.3.0 requires ext-pcntl * -> the requested PHP extension
 pcntl is missing from your system.
    - Installation request for laravel/horizon ^3.3 -> satisfiable by laravel/ho
rizon[v3.3.0, v3.3.1, v3.3.2, v3.4.0, v3.4.1, v3.4.2, v3.4.3, v3.4.4, v3.4.5, v3
.4.6, v3.4.7, v3.5.0, v3.6.0, v3.6.1, v3.7.0, v3.7.1, v3.7.2].

  To enable extensions, verify that they are enabled in your .ini files:
    - D:\xampp\php\php.ini
  You can also run `php --ini` inside terminal to see which files are used by PH
P in CLI mode.

**Comment by trwnh** [comment]
so enable those extensions

**Comment by ER3777** [comment]
I had enabled all the extensions post which i again get the below error, when trying to run this command **php artisan key:generate**

D:\xampp\htdocs\pixelfed>php artisan key:generate
PHP Fatal error:  Uncaught Error: Class 'Illuminate\Foundation\Application' not
found in D:\xampp\htdocs\pixelfed\bootstrap\app.php:14
Stack trace:
#0 D:\xampp\htdocs\pixelfed\artisan(20): require_once()
#1 {main}
  thrown in D:\xampp\htdocs\pixelfed\bootstrap\app.php on line 14

Fatal error: Uncaught Error: Class 'Illuminate\Foundation\Application' not found
 in D:\xampp\htdocs\pixelfed\bootstrap\app.php:14
Stack trace:
#0 D:\xampp\htdocs\pixelfed\artisan(20): require_once()
#1 {main}
  thrown in D:\xampp\htdocs\pixelfed\bootstrap\app.php on line 14

**Comment by dansup** [comment]
@ER3777 Have you tried running ```composer dump-autoload -o``` ?
